lib/utils/conf_gen.py

```diff
@@ -530,7 +551,10 @@ def _helper(
 
             # Handles ldap scenarios with configs that allow commas in the config and
             # are returned with commas. e.g. "query-base-dn" and "user-dn-pattern"
-            if len(split_value) > 1 and not ("-dn" in config and "ldap" in context):
+            if len(split_value) > 1 and not (
+                ("-dn" in config and "ldap" in context)
+                or config in {"dcs", "namespaces"}
```

Comment:
What is the `or config in {"dcs", "namespaces"}` handling?

Reply:
The info responses return `dcs=DC1,DC2` and I want to make sure we don't split those. Once they are split, and turned into a list it is more difficult to remove them later in the step that deletes invalid keys since `dcs` is an invalid key